backend/internal/cert/system_cert_service.go

```diff
@@ -29,22 +29,28 @@ import (
 	"github.com/asgardeo/thunder/internal/system/crypto/hash"
 )
 
+// TODO; Need to identify the usage properly and remove if not required.
```

Comment:
Let's remove this comment and track in the git issue

Reply:
Fixed